docs/workload-development-kit/workload-environment.md

```diff
@@ -50,6 +48,33 @@ During the development cycle, testing a workload on a nonproduction tenant can b
 
 :::image type="content" source="./media/workload-environment/cloud-mode-diagram.png" alt-text="Diagram of the cloud mode architecture." lightbox="./media/workload-environment/cloud-mode-diagram.png":::
 
+### Workload NuGet package structure
+
+The workload is packaged as a NuGet package, combining backend and frontend components. The structure adheres to specific naming conventions and is enforced by Fabric for consistency across upload scenarios.
+The NuGet package designed to represent workloads is structured to include both backend and frontend components.
+
+#### Backend Structure
+
+The backend segment comprises XML files that define the Workload and its associated Items, which are essential for registration within the Fabric system.
+
+##### Key Components:
+- `WorkloadManifest.xml` - The workload configuration file, required to have this exact name for Fabric's verification.
```

Comment:
Suggest adding here a reference to the docs related to each manifest (the original Packages/manifest README in boilerplate)

Reply:
P2 improvement, in a separate PR